Source/Core/UICommon/DiscordPresence.cpp

```diff
@@ -82,6 +87,30 @@ static void HandleDiscordJoin(const char* join_secret)
 
   event_handler->DiscordJoin();
 }
+
+std::string ArtworkForGameId(const std::string& gameid)
+{
+  static const std::set<std::string> REGISTERED_GAMES{
+      "GALX01",
+  };
+
+  std::string region_neutral_gameid = gameid;
+  if (region_neutral_gameid.size() >= 4)
+  {
+    region_neutral_gameid[3] = 'X';
+  }
+
+  if (REGISTERED_GAMES.count(region_neutral_gameid) != 0)
```

Comment:
What about `REGISTERED_GAMES.contains(region_neutral_gameid)`?

Reply:
That doesn't exist yet :)